services/QuillLMS/client/app/bundles/Teacher/components/salesForm/lowerFormFields.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+import { Input } from '../../../Shared';
+import { SCHOOL_PREMIUM_ESTIMATE, TEACHER_PREMIUM_ESTIMATE, STUDENT_PREMIUM_ESTIMATE, COMMENTS } from '../../../../constants/salesForm';
+
+export const LowerFormFields = ({ errors, handleUpdateField, schoolPremimumEstimate, teacherPremimumEstimate, studentPremimumEstimate, comments }) => {
```

Comment:
Looks like there's a typo in all the estimate fields here (`Premimum` instead of `Premium`)